tests/testthat/test-get-info-covidregionaldata.R

```diff
@@ -0,0 +1,26 @@
+# Test get_info_covidregionaldata
+test_that("get_info_covidregionaldata returns dataframe", {
+  countries <- get_info_covidregionaldata() %>%
+    dplyr::filter(get_data_function == "get_regional_data" & !is.na(source_data_cols))
+  
+  expect_s3_class(countries, "data.frame")
+})
+
+test_that("get_info_covidregionaldata returns at least 13 countries", {
+  countries <- get_info_covidregionaldata() %>%
+    dplyr::filter(get_data_function == "get_regional_data" & !is.na(source_data_cols))
+  
+  n_countries <- nrow(countries)
+  
+  expect_gte(n_countries, 13)
```

Comment:
I've never seen this and can't guess at what gte stands for but I guess :+1:?

Reply:
greater than or equal to - it's the new version of `expect_more_than` - agree it's a bit too concise!

Reply:
nice!